pom.xml

```diff
@@ -91,6 +92,11 @@
             <artifactId>echarts-api</artifactId>
             <version>${echarts-api.version}</version>
         </dependency>
+        <dependency>
```

Comment:
This should make 
```
       <dependency>
            <groupId>org.jenkins-ci.plugins</groupId>
            <artifactId>scm-api</artifactId>
        </dependency>
        <dependency>
            <groupId>org.jenkins-ci.plugins</groupId>
            <artifactId>branch-api</artifactId>
        </dependency>
```
obsolete. Remove and check if there are still references to those classes.

Reply:
No, there are not references anymore.